highfidelity/hifi
# Feat/normal map compression

Adds support for compressed normal maps.

 ## Test plan:
- Go to `Developer > Graphics...`
- Turn on "Compress Normal Textures"
- Shutdown Interface
- Clean cache
- Verify normal maps are correct. (Wooden floor, coffee cups in Welcome, Rock floor in playa, sand in localhost/garden)
- Standard graphics smoke test
(Ping me on Slack so I can guide you through doing that if needed)
